src/common/base/Cord.cpp

```diff
@@ -112,8 +112,9 @@ size_t Cord::appendTo(std::string& str) const {
     }
 
     // Last block
-    str.append(tail_, blockPt_);
-
+    if (tail_) {
```

Comment:
the tail_ maybe nullptr?

Reply:
I am confused in which case the tail_ will be nullptr.

Reply:
Haha, I finally found out why.
When the pr created,  rows 101 and 102  in this file did not exist
Reference # 950
now this case not exitsbin. 

👍👍👍

Reply:
fix